packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1299,24 +1190,28 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     }
   }
 
-  function scheduleErrorRecovery(fiber: Fiber) {
-    scheduleWorkImpl(fiber, Sync, true);
-  }
-
   function recalculateCurrentTime(): ExpirationTime {
     // Subtract initial time so it fits inside 32bits
-    const ms = now() - startTime;
-    mostRecentCurrentTime = msToExpirationTime(ms);
+    mostRecentCurrentTimeMs = now() - originalStartTimeMs;
+    mostRecentCurrentTime = msToExpirationTime(mostRecentCurrentTimeMs);
     return mostRecentCurrentTime;
   }
 
   function deferredUpdates<A>(fn: () => A): A {
     const previousExpirationContext = expirationContext;
-    expirationContext = computeAsyncExpiration();
+    const previousUpdatesAreInteractive = updatesAreInteractive;
+    const currentTime = recalculateCurrentTime();
+    expirationContext = computeAsyncExpiration(currentTime);
+    // We assume any use of `deferredUpdates` is to split a high-priority
+    // interactive update into separate high and low values. Set this to true so
+    // that the low-pri value is treated as if it were the result of an
+    // interaction, even if we're in the render phase.
```

Comment:
I don't understand this, can you explain more? Does this fix the issue I reported or is it unrelated?

Reply:
Meant to delete this, not needed